managed-hubs/sales.md

```diff
@@ -7,56 +7,86 @@ This page has information to help guide this process.
+Status fields indicate each step of the sales process
+: We use Monday **status fields** to indicate where we are in the leads process. For example, a new lead is in the "Send Questionnaire" status of the "Triage Phase". This means that the next action is to send them a questionnaire.
 
-Each lead gets a Drive folder
-: Once we create any kind of asset attached to a lead (e.g., a budget, SOW, or notes from a meeting), create a folder for them. These exist in [the `sales/leads/` folder of the 2i2c Google Drive](https://drive.google.com/drive/folders/1zBXL0X3S28W6iNAshBMaXF7gSpvnXa83?usp=sharing).
+Fill in information following each status column
+: As leads progress through the sales process, we need more information from them, eventually leading to a contract. Fill in this information, and once we reach the next "status" column on the board, we have entered a new phase of the process. At the end of a sale (AKA, once an invoice is sent an paid), all columns of the row should be filled.
 
-### Converting leads into sales
+### Triage stage
 
-The leads questionnaire helps triage and estimate work
-: We gather the most structured and useful information for our leads using [the leads generation questionnaire](https://docs.google.com/forms/d/1KHw-4Wdyoofv-6CENeiOSbnC62LKdx55QTcZmXQVkkc/edit?usp=sharing). This should have questions that allow us to gather the information noted in [](sales:questions-to-answer).
+The first stage is to understand whether an interested party is in-scope for 2i2c, and a good fit for the Managed Hub Service.
+
+Create a lead
+: When others show obvious interest in working with 2i2c, they are now a lead. [**Create a new item in the sales pipeline**](https://icsi-company.monday.com/boards/1140216585) and fill in the relevant information for them (everything up to "triage status").
+
+Send the questionnaire
+: To understand whether a lead is a good fit, we've created [a short questionnaire](https://forms.gle/VFzs364iNrJgKnKu9) for them. Send a link to the questionnaire.
+
+Create a Lead Folder for them
+: As soon as a lead responds to the questionnaire, create a folder for them [in the leads folder](https://drive.google.com/drive/folders/1msuG0xWPbawMwesD8LZxdMClP-GVpgT2?usp=sharing). Use this to store any lead-specific information, such as price quotes and contracts.
+
+Decide if they are in-scope
+: If the lead is in-scope, then set the Triage status column to "In Scope" and move on to the negotiation stage.
+
+### Negotiation stage
+
+The negotiation stage is where we understand a bit more about what the lead needs and whether we'll be able to provide it. For example, do they want something really custom that goes beyond our standard Managed Hub distributions? If so, they might better as a "development lead" rather than a "managed service lead".
```

Comment:
```suggestion
The negotiation stage is where we understand a bit more about what the lead needs and whether we'll be able to provide it. For example, do they want something really custom that goes beyond our standard Managed Hub distributions? If so, they might be a "development lead" rather than a "managed service lead".
```